### PR TITLE
fix(social-feed): make less api calls

### PR DESCRIPTION
**Pull request by WaDadidou**
https://github.com/TERITORI/teritori-dapp/issues/522

### CODE REVIEW

File: packages/hooks/feed/useUpdateAvailableFreePost.ts [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-  const [freePostCount, setFreePostCount] = useState(0);
-  const updateAvailableFreePost = useCallback(
-    async (networkId: string, postCategory: PostCategory, wallet?: Wallet) => {
-      const freePost = await getAvailableFreePost({
+export const useUpdateAvailableFreePost = (
+  networkId: string,
+  postCategory: PostCategory,
+  wallet?: Wallet
+) => {
+  const { data } = useQuery(
```
Comment (hthieu1110) [inline packages/hooks/feed/useUpdateAvailableFreePost.ts:12]: We should add ```
{
      staleTime: Infinity,
    }
```

File: packages/hooks/feed/useUpdatePostFee.ts [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
-      setPostFee(fee || 0);
-    },
-    []
-  );
-  return { postFee, updatePostFee };
+export const useUpdatePostFee = (
+  networkId: string,
+  postCategory: PostCategory
+) => {
+  const { data } = useQuery([networkId, postCategory], async () => {
```
Comment (hthieu1110) [inline packages/hooks/feed/useUpdatePostFee.ts:10]: We should add { staleTime: Infinity, }

File: packages/screens/DAppStore/components/DAppStoreData.tsx [hunk 3]
```diff
@@ -57,6 +57,7 @@ export const DAppStoreData: React.FC = memo(() => {
     };
 
     record.options.forEach(function (option: string) {
+      if (!dAppsCol[option]) return;
```
Comment (hthieu1110) [inline packages/screens/DAppStore/components/DAppStoreData.tsx:60]: maybe we can. assign `dAppsCol[option]` to a var to make the code more readable

**Review by n0izn0iz — DISMISSED** [review]
LGTM

**Review by hthieu1110 — DISMISSED** [review]
LGTM

**Review by hthieu1110 — APPROVED** [review]
LGTM

**Comment by zxxma** [comment]
LGTM 👍

**Pull request merged**